examples/practice-typescript/src/components/SideBarMenuItem/SideBarMenuItem.tsx

```diff
@@ -0,0 +1,19 @@
+import './sideBarMenuItem.scss';
+
+interface SideBarMenuItemProps {
+  text: string;
+  router: string;
+  status?: 'aside__active' | '';
```

Comment:
What is this?

Reply:
This one should be a boolean `isActive` instead

Reply:
Fixed